po/es.po

```diff
@@ -68,7 +68,7 @@ msgstr "Preferencias"
 #: src/Widgets/Headerbar.vala:56
 #, fuzzy
 msgid "Show an indicator:"
-msgstr "Mostrar icone de panel"
+msgstr "Mostrar un icono en el panel"
```

Comment:
I don't understand Spanish, but is it intentional that there is no trailing colon that the source string has?

Reply:
Thank you for your suggestion!